WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift

```diff
@@ -172,116 +131,10 @@ class MediaLibraryViewController: UIViewController {
-
-        wrapSearchBarInPaddingView()
-
-        let height = searchBar.intrinsicContentSize.height
-        let heightConstraint = searchBarContainer.heightAnchor.constraint(equalToConstant: height)
-        heightConstraint.priority = UILayoutPriorityDefaultLow
-        heightConstraint.isActive = true
-
-        let expandedHeightConstraint = searchBarContainer.heightAnchor.constraint(greaterThanOrEqualToConstant: height)
-        expandedHeightConstraint.priority = UILayoutPriorityRequired
-        expandedHeightConstraint.isActive = true
-
-        searchBarContainer.layoutIfNeeded()
-        searchBar.sizeToFit()
-    }
-
-    private func wrapSearchBarInPaddingView() {
-        let paddingView = UIView()
-        paddingView.translatesAutoresizingMaskIntoConstraints = false
-        searchBarContainer.addSubview(paddingView)
-        paddingView.addSubview(searchBar)
-
-        var leading = searchBarContainer.leadingAnchor
-        var trailing = searchBarContainer.trailingAnchor
-
-        if #available(iOS 11.0, *) {
-            leading = searchBarContainer.safeAreaLayoutGuide.leadingAnchor
-            trailing = searchBarContainer.safeAreaLayoutGuide.trailingAnchor
-        }
-
-        NSLayoutConstraint.activate([
-            paddingView.leadingAnchor.constraint(equalTo: leading),
-            paddingView.trailingAnchor.constraint(equalTo: trailing),
-            paddingView.topAnchor.constraint(equalTo: searchBarContainer.topAnchor),
-            paddingView.bottomAnchor.constraint(equalTo: searchBarContainer.bottomAnchor),
-            ])
-    }
-
-    private func addNoResultsView() {
-        guard let noResultsView = WPNoResultsView(title: nil,
-                                               message: nil,
-                                               accessoryView: UIImageView(image: UIImage(named: "media-no-results")),
-                                               buttonTitle: nil) else { return }
-
-        pickerViewController.collectionView?.addSubview(noResultsView)
-        noResultsView.centerInSuperview()
-
-        noResultsView.delegate = self
-
-        self.noResultsView = noResultsView
-    }
-
-    // MARK: - Keyboard handling
-
-    private func registerForKeyboardNotifications() {
-        NotificationCenter.default.addObserver(self, selector: #selector(keyboardDidChangeFrame(_:)), name: NSNotification.Name.UIKeyboardDidChangeFrame, object: nil)
-    }
-
-    private func unregisterForKeyboardNotifications() {
-        NotificationCenter.default.removeObserver(self, name: NSNotification.Name.UIKeyboardDidChangeFrame, object: nil)
-    }
-
-    @objc private func keyboardDidChangeFrame(_ notification: Foundation.Notification) {
-        let duration = notification.userInfo?[UIKeyboardAnimationDurationUserInfoKey] as? TimeInterval ?? 0.2
-
-        UIView.animate(withDuration: duration) {
-            self.noResultsView?.centerInSuperview()
+        if self.searchBar?.isFirstResponder == .some(true) {
```

Comment:
You don't need `self.` here, and you can also leave out the `.some` – just `if ... == true`